pyscf/gw/rpa.py

```diff
@@ -39,14 +39,14 @@
 # core routines, kernel, rpa_ecorr, rho_response
 # ****************************************************************************
 
-def kernel(rpa, mo_energy, mo_coeff, Lpq=None, nw=None, verbose=logger.NOTE):
+def kernel(rpa, mo_energy, mo_coeff, Lpq=None, nw=None, x0=None, verbose=logger.NOTE):
```

Comment:
`x0=0.5` by default ?

Reply:
x0=0.5 is the default value in the corresponding grid generation subroutine [def _get_scaled_legendre_roots(nw, x0=0.5)] and probably needs to be kept at that value to avoid breaking test calculations.

Reply:
That's what I mean. I guess I didn't make a clear statement. Since the subsequent function calls do not test if x has a valid value, I recommend x0=0.5 instead of x0=None here as the default value.

Reply:
I added default values in kernels for both nw and x0 in rpa.py and urpa.py